### PR TITLE
Update doc for the clearing of meta-cache

### PR DESCRIPTION
**Pull request by SeanSong25**
This commit introduces a new command called clear_server_metacache which helps the user to delete all the meta-cache stored on a tablet server.

### CODE REVIEW

File: docs/content/preview/admin/yb-ts-cli.md
Comment (ddorian) [inline docs/content/preview/admin/yb-ts-cli.md:276]:
```suggestion
##### clear_server_metacache
```

File: docs/content/preview/admin/yb-ts-cli.md
Comment (ddorian) [inline docs/content/preview/admin/yb-ts-cli.md:277]: Please add a description and use the same format as the other commands. Explain if it works on both yb-tserver/master, and 1-2 sentence what metacache is.

**Review by ddorian — CHANGES_REQUESTED** [review]

File: docs/content/preview/admin/yb-ts-cli.md
Comment (SeanSong25) [reply docs/content/preview/admin/yb-ts-cli.md:284]: Ok, Will do

**Review by ddorian — COMMENTED** [review]
Ok by me. Please wait for Mark/Dwight approval.

**Comment by mdbridge** [comment]
it looks like the index  pointing to each of the subcommands does not have the new command:
Commands
The following commands are available:

[are_tablets_running](https://deploy-preview-21684--infallible-bardeen-164bc9.netlify.app/preview/admin/yb-ts-cli/#are-tablets-running)
[is_server_ready](https://deploy-preview-21684--infallible-bardeen-164bc9.netlify.app/preview/admin/yb-ts-cli/#is-server-ready)
[compact_all_tablets](https://deploy-preview-21684--infallible-bardeen-164bc9.netlify.app/preview/admin/yb-ts-cli/#compact-all-tablets)


 I assume it should be updated to point to the new command

**Comment by mdbridge** [comment]
(for reference, the preview is currently at https://deploy-preview-21684--infallible-bardeen-164bc9.netlify.app/preview/admin/yb-ts-cli/)

**Comment by mdbridge** [comment]
it looks to me like the commands are supposed to be listed in alphabetical order; please put the new command in the appropriate place
 
 documentation folks, it looks like the is_server_ready subcommand  in that list is out of order; you may want to fix it

**Comment by mdbridge** [comment]
"Works on both YB-Master (port 9100) and YB-TServer (port 7100) process": process should be plural here
 "No parameters needed.": you do need the server_address parameter, do you not?  I think you can just omit this

**Review by mdbridge — CHANGES_REQUESTED** [review]
"Metacache is a YBClient cache on a Tablet/Master server to find out which Tablet Server hosts which tablet.": ... which is used to find out...

"This cache can be stale in some cases, this

command is introduced for users to clear the metacache on a particular YB-TServer or YB-Master."
-> Because this cache..., this command is provided...

 meta-cache has a -
the meta-cache*s*

"The host and port of the tablet server": could also be a master server
...  of a particular TServer or master

File: docs/content/preview/admin/yb-ts-cli.md
Comment (ddhodge) [inline docs/content/preview/admin/yb-ts-cli.md:87]: What is YBClient? @mdbridge

Comment (ddhodge) [inline docs/content/preview/admin/yb-ts-cli.md:87]:
```suggestion
Metacache is a YBClient cache on a Tablet/Master server which is used to find out which tablet server hosts which tablet. Because this cache can be stale in some cases, you can use this command to clear the metacache on a particular YB-TServer or YB-Master.
```

File: docs/content/preview/admin/yb-ts-cli.md
Comment (ddhodge) [inline docs/content/preview/admin/yb-ts-cli.md:89]:
```suggestion

**Syntax**
```

File: docs/content/preview/admin/yb-ts-cli.md
Comment (SeanSong25) [reply docs/content/preview/admin/yb-ts-cli.md:87]: YBClient is a client class living inside a T-Server, it is what the users talk to when they connect to a tablet server. It has many usage, but as an example, in our case, when a T-Server received a write request from the user, the YBClient will use its meta-cache to figure out which tablet this request needs to write to, and if the T-Server itself does not host that tablet, it will go talk to master to find out what tablet it should issue the request to, cache this information in the meta-cache and retry the RPC request to the correct T-Server. @mdbridge please correct me or add more information if I am missing anything. @ddhodge Do you think I should add more detailed info for this command?

File: docs/content/preview/admin/yb-ts-cli.md
Comment (SeanSong25) [reply docs/content/preview/admin/yb-ts-cli.md:87]: Hi Dwight, why do I need to add one more line of spacing here?

File: docs/content/preview/admin/yb-ts-cli.md
Comment (mdbridge) [reply docs/content/preview/admin/yb-ts-cli.md]: I agreed that YBClient  is probably a layer of detail too far.

Maybe something closer to:

Tablet servers and masters use MetaCache's to cache information about which tablet server hosts which tablet.  Because these caches could become stale in some cases, you may want to use this command to clear the MetaCaches on a particular tablet server or master.

File: docs/content/preview/admin/yb-ts-cli.md
Comment (ddhodge) [reply docs/content/preview/admin/yb-ts-cli.md:87]: My mistake, I intended to put a space below, not above. It's just nicer formatting

File: docs/content/preview/admin/yb-ts-cli.md
Comment (ddhodge) [reply docs/content/preview/admin/yb-ts-cli.md]: I wouldn't add any description of ybclient or more detail, no. ybclient doesn't appear (well, hardly) in the docs elsewhere, so that's why I was asking. mbridge's suggestion seems good.

**Review by ddhodge — APPROVED** [review]

**Review by mdbridge — APPROVED** [review]
seem satisficing to me.

**Comment by mdbridge** [comment]
@ddhodge let @SeanSong25 know if you want the YBCLient references removed

File: docs/content/preview/admin/yb-ts-cli.md
Comment (SeanSong25) [reply docs/content/preview/admin/yb-ts-cli.md]: Just committed ro remove YBClient reference, please check if this satisfies the requireent

Comment (ddhodge) [reply docs/content/preview/admin/yb-ts-cli.md]: LGTM

**Comment by mdbridge** [comment]
looks like @SeanSong25 fixed the YBClient references

File: docs/content/preview/admin/yb-ts-cli.md
Comment (ddhodge) [reply docs/content/preview/admin/yb-ts-cli.md]: @SeanSong25 what versions does this apply to? Only 2.21?

Comment (SeanSong25) [reply docs/content/preview/admin/yb-ts-cli.md]: Yes, I've only backported it to 2024.1, and not anything before that

Comment (SeanSong25) [reply docs/content/preview/admin/yb-ts-cli.md]: can I squash and merge this?

Comment (ddhodge) [reply docs/content/preview/admin/yb-ts-cli.md]: If by that you mean it only applies to preview, then yes, please merge, and thanks

**Comment by mdbridge** [comment]
@ddorian did you still want Sean to change something?

**Comment by ddorian** [comment]
@mdbridge I'm good. My review was stuck in requesting changes.

**Pull request merged**
